lib/module.js

```diff
@@ -735,7 +736,7 @@ Module._initPaths = function() {
     paths.unshift(path.resolve(homeDir, '.node_modules'));
   }
 
-  var nodePath = process.env.NODE_PATH;
+  var nodePath = safeGetenv('NODE_PATH');
```

Comment:
here I'm touching windows @addaleax and indeed windows tests are failing. 👍 
I see the value of changing `NODE_PATH` in a test, 
Ill consider windows vs Unix on that too, Is it ok?